devel/release.md

```diff
@@ -15,7 +15,17 @@ report issues. Typically, there is a single alpha or beta release, but if there
 are a higher than expected number of issues there can be multiple releases
 (e.g, a2 or b2).
 
-## Create or update release branch
+## 1. Update submodules
+
+Update submodules by running the following script:
```

Comment:
could you link to https://github.com/kubernetes-client/python/blob/master/devel/submodules.md#update-submodule instead? In that way we make sure the instructions don't diverge in the future

Reply:
Fixed.